### PR TITLE
Fix usernet socket name too long

### LINKED ISSUE #1678: usernet: socket name is too long
`len("/Users/<USER>/.lima/<INSTANCE>/usernet_<NETWORK>_endpoint.sock")` may hit the 104 limit

https://github.com/lima-vm/lima/blob/5a3504432866ecc3f93e459bc1f9a97f846e1f5c/pkg/networks/usernet/config.go#L27-L30

### PR DESCRIPTION
**Pull request by balajiv113**
fixes #1678

### CODE REVIEW

File: pkg/networks/usernet/config_test.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+	t.Run("max network path in instance directory", func(t *testing.T) {
+		limaHome, err := dirnames.LimaDir()
+		assert.NilError(t, err)
+		longestInstDir := filepath.Join(limaHome, filenames.LongestSock)
+		_, err = SockWithDirectory(longestInstDir, filenames.LongestSock, QEMUSock)
+		assert.NilError(t, err)
+	})
+
+	t.Run("max network path in _networks directory", func(t *testing.T) {
+		instName := filenames.LongestSock
```
Comment (AkihiroSuda) [inline pkg/networks/usernet/config_test.go:22]: Maybe:
```suggestion
		instName := "a" * len(filenames.LongestSock)
```

File: pkg/networks/usernet/config.go [hunk 2]
```diff
@@ -4,6 +4,8 @@ import (
 	"fmt"
 	"path/filepath"
 
+	"github.com/lima-vm/lima/pkg/osutil"
+
```
Comment (AkihiroSuda) [inline pkg/networks/usernet/config.go:8]: This empty line can be removed

File: pkg/networks/usernet/config_test.go (hunk 1 above)
Comment (balajiv113) [reply pkg/networks/usernet/config_test.go:22]: You mean this ?

```
strings.Repeat("a", len(filenames.LongestSock))
```

Comment (AkihiroSuda) [reply pkg/networks/usernet/config_test.go:22]: Yes 😅

Comment (jandubois) [reply pkg/networks/usernet/config_test.go]:
```
$ perl -E 'say "a" x 20'
aaaaaaaaaaaaaaaaaaaa
```

😹

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**